drivers/common/opensensors.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __OPENSENSORS_H__
+#define __OPENSENSORS_H__
+
+#define MAXSENSORS           10
+
+typedef uint16_t (*callbackRead_cbt)(void);
+
+typedef float (*callbackConvert_cbt)(uint16_t value);
+
+typedef struct {
```

Comment:
There's the need to match a generic sensor hardware to some standard CoAP sensor resource. For instance, we have a temperature sensor on a sht21, but the CoAP resource created should be /s/t. How can we match a hardware chip to a "standard" CoAP resource? This association is done in sensors.c, the board-specific file in the framework. Should we introduce intermediate IDs in opendefs.h (e.g., sensors.c will match the sht21 hardware with a temperatureID and than csensors will match the temperatureID with its CoAP path1)?